src/lib/CoreInstance.cc

```diff
@@ -280,8 +266,10 @@ void CoreInstance::createCore() {
 
 void CoreInstance::createSpecialFileDirectory() {
   // Create the Special Files directory if indicated to do so in Config
-  if (config_["CPU-Info"]["Generate-Special-Dir"].as<bool>() == true) {
-    simeng::SpecialFileDirGen SFdir = simeng::SpecialFileDirGen(config_);
+  bool generate;
+  config_["CPU-Info"]["Generate-Special-Dir"] >> generate;
```

Comment:
Can' this be replaced with `config::SimInfo::getGenSpecFiles()`?

Reply:
Yes, good spot